client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/theme-info-popup.tsx

```diff
@@ -35,8 +35,13 @@ const ThemeInfoPopup = ( { slug }: ThemeInfoPopupProps ) => {
 			<div className="theme-info-popup__support-links popup-item">
 				<h2>Support</h2>
 				<p>
-					<a href="https://wordpress.com/help/contact/">Contact us</a> or visit the{ ' ' }
-					<a href="https://en.forums.wordpress.com/forum/themes">support forum</a>
+					<a href="https://wordpress.com/help/contact/" target="__blank">
```

Comment:
```suggestion
					<a href="https://wordpress.com/help/contact/" target="__blank" rel="noopener noreferrer">
```

Reply:
It may have changed but my understanding was, as of last year, this was still an issue with Safari and a couple of other browsers.

But happy to ignore the suggestion if this is now default behavior everywhere!

Reply:
Double-checked and it looks like all the major browsers support this. Thanks for double-checking!